### PR TITLE
Theme submission "Lostkeys"

### PR DESCRIPTION
**Pull request by lostkeys**
A dark and light theme suitable for note taking and documentations

### CODE REVIEW

File: CNAME [hunk 1]
```diff
@@ -1 +0,0 @@
-theme.typora.io
```
Comment (abnerlee) [inline CNAME]: Don't need to change this

File: _config.yml [hunk 2]
```diff
@@ -25,3 +25,5 @@ kramdown:
 
 
 exclude: ['README.md', 'Gemfile.lock', 'Gemfile', 'Rakefile', 'vendor']
+
+theme: jekyll-theme-minimal
```
Comment (abnerlee) [inline _config.yml]: Don't need to change this either

**Comment by lostkeys** [comment]
Fixed issues with PR

**Comment by abnerlee** [comment]
Thanks for the new theme, looks great.

BTW, you could use and overwrite exisiting CSS variables in https://github.com/typora/typora-theme-toolkit/blob/master/html-preview/theme/default/base-control.css

For example rename `--background-color` to `--bg-color` directly, 
or, to reduce your changes, add

```
--bg-color: var(--background-color)
--text-color: var(--font-color);
...
```

It will bring more compatibility, in case your theme did not over all components. 

We will also update your doc for custom themes recently.

**Pull request merged**